minecraft_servers/timolia/manifest.json

```diff
@@ -5,6 +5,10 @@
   "server_wildcards": [
     "%.timolia.de"
   ],
+  "supported_languages": [
+    "de",
+    "en"
+  ],
```

Comment:
@l3nnartt Obviously I was a bit to slow but doesn't Timolia support more than these two languages?

Reply:
Yes (de/en/es/ru/ua), but to be fair I have to say that the translations are not complete.
I don't know if that matters?!

Reply:
Okay, then just leave it like that. thx!